default/stringtables/en.txt

```diff
@@ -339,11 +339,13 @@ PROJECT_BT_STOCKPILE_SHORT_DESC
 Transfers PP into the imperial stockpile
 
 PROJECT_BT_STOCKPILE_DESC
-'''Transfers PP at the planet where this project is enqueued to the imperial stockpile. Anything enqueued below the transfer project at a supply-connected planet will not be allocated PP.
+'''Transfers PP at the planet where this project is enqueued to the imperial stockpile. You can specify the amount and repetition of the transfer. PP spent on the transfer are added to the imperial stockpile.
 
-Only transfers PP that are available at this planet; PP allocation on planets that are not supply-connected to here will not be affected by this project.
+Only transfers PP that are available at this planet so only PP produced in the same supply-group are used and no PP from the imperial stockpile.
```

Comment:
I don't see why your version is better...?

Reply:
The two versions seem like they mostly differ simply in emphasis, or what aspects are made most explicit.  After reading them multiple times I think I'd have a slightly preference for the newer one, which clarifies that the IS cannot itself be used to give PP to a transfer project.

Reply:
phrasing needs to be improved then.

Reply:
@agrrr3 I'd suggest changing "and no PP from the imperial stockpile" to "(and transfer projects cannot be enabled to themselves draw on the Imperial Stockpile, since that would be a pointless cycle.)"

Reply:
The "so" and the "and" before "no PP" are awkward. Split into 3 sentences? @Dilvish-fo's version is overly wordy.